drivers/include/at.h

```diff
@@ -89,33 +89,33 @@ extern "C" {
 #endif
 
 /**
- * @brief 1st end of line character received (S3 aka CR character for a modem).
+ * @brief default OK reply of an AT device.
  */
-#ifndef AT_RECV_EOL_1
-#define AT_RECV_EOL_1   "\r"
+#ifndef CONFIG_AT_RECV_OK
+#define CONFIG_AT_RECV_OK "OK"
 #endif
 
 /**
- * @brief 1st end of line character received (S4 aka LF character for a modem).
+ * @brief default ERROR reply of an AT device.
  */
-#ifndef AT_RECV_EOL_2
-#define AT_RECV_EOL_2   "\n"
+#ifndef CONFIG_AT_RECV_ERROR
+#define CONFIG_AT_RECV_ERROR "ERROR"
 #endif
+/** @} */
 
 /**
- * @brief default OK reply of an AT device.
+1st end of line character received (S3 aka CR character for a modem).
```

Comment:
Why remove the `@brief`? (here and bellow)

Reply:
I shall restore it.
The reason why I removed it was that doxygen documentation looked same and the @brief usage is not consistent in the file.